app/models/period.rb

```diff
@@ -181,6 +185,17 @@ def to_s(format = :mon_year)
     end
   end
 
+  # Returns a Hash with various Period related dates for eash consumption by the view
+  def to_hash
```

Comment:
`def info`? This is returning some view specific dates, `to_hash` suggests that this is a hash representation of the `Period` object.

Reply:
Yeah I struggled on the name for this.  `info` feels a little too general...maybe `full_info` ? I dunno. We can always rename away from `to_hash` if we have a more specific representation of hash we want to add in the future.